public/scripts/main.js

```diff
@@ -92,10 +92,11 @@ function handle(e) {
     ' code=' + e.code
 
   if(e.code.toLowerCase()=="space"){
-    $("#space").toggleClass("pressed");
+    document.querySelector('#space').classList.toggle("pressed");
+    //document.querySelector('#space').classList.toggle('pressed',e.code.toLowerCase()=="space")
```

Comment:
Please remove this commented line from the PR.

Reply:
Ohh sorry I forgot that I commented on it